src/visitor/mssql.rs

```diff
@@ -1108,7 +1099,7 @@ mod tests {
         let (sql, params) = Mssql::build(query).unwrap();
 
         assert_eq!(expected_sql, sql);
-        assert_eq!(vec![Value::integer(0), Value::integer(10)], params);
+        assert_eq!(vec![Value::int32(0), Value::int64(10)], params);
```

Comment:
Since `.limit(usize)` takes an `usize`, it is always coerced as a `Value::Int64`.